diff --git a/src/main/scala/firrtl/Driver.scala b/src/main/scala/firrtl/Driver.scala
index 66e09e8..6a813d8 100644
--- a/src/main/scala/firrtl/Driver.scala
+++ b/src/main/scala/firrtl/Driver.scala
@@ -66,7 +66,7 @@ object Driver {
     */
   @deprecated("Use side-effect free getAnnotation instead", "1.1")
   def loadAnnotations(optionsManager: ExecutionOptionsManager with HasFirrtlOptions): Unit = {
-    val msg = "Driver.loadAnnotations is deprecated, use Driver.getAnnotations instead"
+    val msg = "Driver.loadAnnotations is deprecated, use Driver.getAnnotations instead "
     Driver.dramaticWarning(msg)
     optionsManager.firrtlOptions = optionsManager.firrtlOptions.copy(
       annotations = Driver.getAnnotations(optionsManager).toList
